File: Torneig_Mort_Spring/src/main/java/org/eljust/DTO/JugadorSimpleDTO.java
package org.eljust.DTO;

import lombok.Getter;
import lombok.Setter;

/**
 * Aquest JugadorDTO el fem servir per a llistar tots els jugadors, crear un
 * jugador nou o bé actualitzar un jugador.
 */
@Getter
@Setter
public class JugadorSimpleDTO {

	private Long idJugador;
	private String nom;
	private int edat;
	private boolean sancionat;

}

File: app_torneig_flutter/lib/models/jugador_simple_dto.dart
class JugadorSimpleDTO {
  int? id;
  String? nom;
  int? edat;
  bool? esSancionat;

  JugadorSimpleDTO({
    required this.id,
    this.nom,
    this.edat,
    this.esSancionat,
  });

  JugadorSimpleDTO.fromJSON(Map<String, dynamic> objecteJSON) {
    id = objecteJSON['idJugador'];
    nom = objecteJSON['nom'] ?? "";
    edat = objecteJSON['edat'];
    esSancionat = objecteJSON['sancionat'];
  }
}
